### ISSUE TITLE
Qclash is runing? 

### ISSUE THREAD

**Issue by liuqimintb** [mantech2045/qclash#1]
OS :Ubuntu20.04

liuqimin@liuqimin-laptop:~$ cd /home/liuqimin/Downloads/qclash2
liuqimin@liuqimin-laptop:~/Downloads/qclash2$ sudo ./qclash.sh
[sudo] password for liuqimin: 
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
"/home/liuqimin/Downloads/qclash2"
stateChanged , QProcess::Starting
stateChanged , QProcess::Running
started
refreshTray

**Comment by mantech2045** [comment]
Yes, with this log is printed, indicating that the program is already running in the background, you can see an icon in the system tray, click it to display the main ui.
BTW: u dont't really need to run it with `sudo`

**Comment by liuqimintb** [comment]
Thanks for your replay. I can find this icon now. and more problem,after import proxy node,this node chcked in qclash is acting,But the browser is can not connected to the network through a proxy. Does the system's network interface need to be reset?

**Comment by mantech2045** [comment]
u need to double click yr node, and u will get a notification.

after that u properly need to set yr browser use the proxy, some like below.
```
export http_proxy='http://127.0.0.1:7890'
export htts_proxy='http://127.0.0.1:7890'
```
or u can also go to yr system network setting to set a global proxy.

**Comment by houhouhou9998** [comment]
Hello, I get the same log, but I don't get a icon in my left tray in ubuntu18.04.
